Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/model/Tasks.java

```diff
@@ -86,6 +90,15 @@ public Tasks onlyCompleted() {
         return Tasks.from(internalMap().filter(isCompleted()));
     }
 
+    public Tasks withoutTask(final Task task) {
```

Comment:
Fixed in https://github.com/novoda/spikes/pull/130/commits/27928b970290dc85b8eede7f23cda93d8c04ce83